### PR TITLE
Make required items public

### PR DESCRIPTION
**Pull request by Ignelios**
Expose APIConfiguration, ApiError and all publishers for request.

### CODE REVIEW

**Review by mikasdo — APPROVED** [review]

File: ios/MaasCore/Sources/MaasCore/Api/Utils/CFlow+Publisher.swift [hunk 1]
```diff
@@ -1,26 +1,26 @@
 import Combine
 
-private func cflowToPublisher<T>(_ cflow: CFlow<ApiResult<T>>) -> AnyPublisher<T, ApiError> {
+public func cflowToPublisher<T>(_ cflow: CFlow<ApiResult<T>>) -> AnyPublisher<T, ApiError> {
```
Comment (justasm) [inline ios/MaasCore/Sources/MaasCore/Api/Utils/CFlow+Publisher.swift:3]: This doesn't need to be public

File: ios/MaasCore/Sources/MaasCore/Api/Utils/ApiKotlinFlowPublisher.swift [hunk 2]
```diff
@@ -1,17 +1,17 @@
 import Combine
 
-class ApiKotlinFlowPublisher<T: AnyObject>: Publisher {
+public class ApiKotlinFlowPublisher<T: AnyObject>: Publisher {
```
Comment (justasm) [inline ios/MaasCore/Sources/MaasCore/Api/Utils/ApiKotlinFlowPublisher.swift:3]: Why does this need to be public? We expose a generic `AnyPublisher`

Comment (Ignelios) [inline ios/MaasCore/Sources/MaasCore/Api/Utils/ApiKotlinFlowPublisher.swift:3]:
```suggestion
class ApiKotlinFlowPublisher<T: AnyObject>: Publisher {
```

File: ios/MaasCore/Sources/MaasCore/Api/Utils/CFlow+Publisher.swift (hunk 1 above)
Comment (Ignelios) [inline ios/MaasCore/Sources/MaasCore/Api/Utils/CFlow+Publisher.swift:3]:
```suggestion
func cflowToPublisher<T>(_ cflow: CFlow<ApiResult<T>>) -> AnyPublisher<T, ApiError> {
```

File: ios/MaasCore/Sources/MaasCore/Api/Utils/ApiKotlinFlowPublisher.swift [hunk 3]
```diff
@@ -2,16 +2,16 @@ import Combine
...
     
-    typealias Output = T
-    typealias Failure = ApiError
+    public typealias Output = T
+    public typealias Failure = ApiError
     
     private let flow: CFlow<ApiResult<T>>
     
-    init(_ flow: CFlow<ApiResult<T>>) {
+    public init(_ flow: CFlow<ApiResult<T>>) {
```
Comment (justasm) [inline ios/MaasCore/Sources/MaasCore/Api/Utils/ApiKotlinFlowPublisher.swift:10]: Not sure what it means in Swift for a class to be internal but its members public 🤷

File: ios/MaasCore/Sources/MaasCore/Api/Utils/CFlow+Publisher.swift [hunk 4]
```diff
@@ -1,26 +1,26 @@
 import Combine
 
-private func cflowToPublisher<T>(_ cflow: CFlow<ApiResult<T>>) -> AnyPublisher<T, ApiError> {
+func cflowToPublisher<T>(_ cflow: CFlow<ApiResult<T>>) -> AnyPublisher<T, ApiError> {
```
Comment (justasm) [inline ios/MaasCore/Sources/MaasCore/Api/Utils/CFlow+Publisher.swift:3]: Should remain `private` 🙂

**Review by justasm — APPROVED** [review]

**Review by justasm — APPROVED** [review]
❤️

**Pull request merged**